planner/core/testdata/integration_suite_out.json

```diff
@@ -4878,13 +4879,13 @@
         "SQL": "desc format = 'brief' select * from t right join (select id-2 as b from t) A on A.b=t.id",
         "Plan": [
           "HashJoin 12487.50 root  right outer join, equal:[eq(test.t.id, Column#25)]",
-          "├─TableReader(Build) 9990.00 root  data:Selection",
```

Comment:
Expected, the current build-side has the smaller row-size which results in a smaller hash-build-cost.